### PR TITLE
Working end of level fail for user who has zero resources

### PR DESCRIPTION
**Pull request by adamstag20**
End Of Level displays telling user to restart level if they go to 0 resources.

### CODE REVIEW

File: Isle of Trust/src/components/EndOfLevel.tsx [hunk 1]
```diff
@@ -78,5 +78,31 @@ export function EndOfLevel({ level, success, mapAgents }: EndOfLevelProps) {
...
+
+    return ReactDom.createPortal( 
+        <div className="popup-container">
+            <div className="overlay"></div>
+            <div className="popup"> 
+                <div className="popup-content">
+                            <h1 className="text-center">Oh No!</h1>
+                            <h2 className="fail">Level {level} Failed</h2>
+                            <h2 className="text-center"> Your player ran out of resources!</h2>
+                            <Link
```
Comment (ViktoriyaPetrova) [inline Isle of Trust/src/components/EndOfLevel.tsx:94]: Maybe change this to "You ran out of resources!"

**Review by ViktoriyaPetrova — APPROVED** [review]
Everything looks good! Just one small suggestion.

**Review by hollan2 — APPROVED** [review]
Looks good!

File: Isle of Trust/src/components/EndOfLevel.tsx (hunk 1 above)
Comment (adamstag20) [reply Isle of Trust/src/components/EndOfLevel.tsx:94]: I will make this change

**Pull request merged**